### PR TITLE
fix typo erroring->erring

### PR DESCRIPTION
**Pull request by EruDev**

### CODE REVIEW

**Review by kinow — APPROVED** [review]
👍

**Comment by EruDev** [comment]
@kinow I submitted another coomit, an invalid link

File: src/test/java/org/apache/commons/collections4/bag/AbstractBagTest.java [hunk 1]
```diff
@@ -125,7 +125,7 @@ public void resetFull() {
 
     //-----------------------------------------------------------------------
     /**
-     * Returns the {@link #collection} field cast to a {@link Bag}.
+     * Returns the {@link Collection} field cast to a {@link Bag}.
```
Comment (kinow) [inline src/test/java/org/apache/commons/collections4/bag/AbstractBagTest.java:128]: That's a link to the [`collection`](https://github.com/apache/commons-collections/blob/3945e6c48e2dd83c6b4b65c135f0bfc2f28ed52a/src/test/java/org/apache/commons/collections4/collection/AbstractCollectionTest.java#L141) variable of the parent class.

**Review by kinow — CHANGES_REQUESTED** [review]

File: src/test/java/org/apache/commons/collections4/bag/AbstractBagTest.java (hunk 1 above)
Comment (EruDev) [reply src/test/java/org/apache/commons/collections4/bag/AbstractBagTest.java]: sorry..I revert AbstractBagTest

Comment (kinow) [reply src/test/java/org/apache/commons/collections4/bag/AbstractBagTest.java]: No problem, thanks @EruDev . Could you squash your commits too? For a small change, it's preferable to have a single (or very few) commit. Please.

Thanks!

Comment (EruDev) [reply src/test/java/org/apache/commons/collections4/bag/AbstractBagTest.java]: I close this PR and resubmit.

**Pull request closed without merging**